templates/azure/pro.html

```diff
@@ -324,6 +335,11 @@ <h2>Get Ubuntu Pro on Azure now:</h2>
     </div>
   </div>
   <div class="row p-divider">
+    <div class="col-3 p-divider__block">
```

Comment:
It might not be necessary but would it look better if all these are in one line? 

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/57550290/193799322-9088a92f-46be-4234-ac28-79165d204fae.png">